framework/validators/UniqueValidator.php

```diff
@@ -133,7 +145,24 @@ public function validateAttribute($model, $attribute)
         }
 
         if ($exists) {
-            $this->addError($model, $attribute, $this->message);
+            if (is_array($targetAttribute)) {
+                $attributeCombo = [];
+                $valueCombo = [];
+                foreach ($targetAttribute as $k => $v) {
```

Comment:
According to [core code style](https://github.com/yiisoft/yii2/blob/master/docs/internals/core-code-style.md#42-properties) we prefer to avoid short variable names such as `$k`, `$v`. Could you rename it, please?